main.py

```diff
@@ -1,10 +1,37 @@
+import json
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+    for user_name in data:
+        user = data[user_name]
+        race, flag = Race.objects.get_or_create(
+            name=user["race"]["name"],
+            description=user["race"]["description"])
```

Comment:
Where do you using `flag` viriable ? You can unpuck it in `_` if you do not using it